core/test/Cardano/Address/ScriptSpec.hs

```diff
@@ -85,31 +85,27 @@ import qualified Data.Text.Encoding as T
 
 spec :: Spec
 spec = do
-    let mnemonic = [ "test", "child", "burst", "immense", "armed", "parrot"
-                   , "company", "walk", "dog" ]
-    let (Right mw) = mkSomeMnemonic @'[9,12,15,18,21,24] mnemonic
+    let mnemonic = ["network","empty","cause","mean","expire","private","finger"
+                   ,"accident","session","problem","absurd","banner","stage","void","what"]
+    let (Right mw) = mkSomeMnemonic @'[15] mnemonic
```

Comment:
Why this change :thinking:  ?

Reply:
oh, was double checking in ghci and used 15-word mnemonic rather than 9-word

Reply:
ok, will return to original mnemonics in coming commit